axonserver/src/main/java/io/axoniq/axonserver/localstorage/TrackingEventProcessorManager.java

```diff
@@ -177,11 +180,16 @@ public void close() {
         private final StreamObserver<InputStream> eventStream;
         private volatile CloseableIterator<SerializedEventWithToken> eventIterator;
         private volatile boolean running = true;
+        private final Set<PayloadDescription> blacklistedTypes = new CopyOnWriteArraySet<>();
+        private volatile int force = blacklistedSendAfter;
 
         private EventTracker(GetEventsRequest request, StreamObserver<InputStream> eventStream) {
             permits = new AtomicInteger((int) request.getNumberOfPermits());
             lastPermitTimestamp = new AtomicLong(System.currentTimeMillis());
             nextToken = new AtomicLong(request.getTrackingToken());
+            if( request.getBlacklistCount() > 0) {
```

Comment:
Super nit: the default space between `if` and `(` should/could be covered by the style guide I think.